### PR TITLE
Disabled unwanted I18n messages

### LINKED ISSUE #1269: How to get rid of "I18n: section was not found in current dictionary" message in console
Is there a way I can get rid of "**I18n: section was not found in current dictionary**" message that is flooding my browser console?

**I do not need i18n support** and I am not sure if there is any way to stop these messages to come up.

image

### PR DESCRIPTION
**Pull request by robonetphy**
- Resolve #1269 
- remove all the warnings using `yarn lint`

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - `Improvements` - The `initialBlock` property of Editor config is deprecated. Use the `defaultBlock` instead. [#993](https://github.com/codex-team/editor.js/issues/993)
 - `Fix` - Fixed the issue of toolbar not disappearing on entering input in Chinese, Hindi and some other languages. [#1196](https://github.com/codex-team/editor.js/issues/1196)
 - `Improvements` - BlockAPI `call()` method now returns the result of calling method, thus allowing it to expose arbitrary data as needed [#1205](https://github.com/codex-team/editor.js/pull/1205)
+- `Improvements` - Disable the debugging messages for section defined or not in I18n module, thus disabling debugging messages on console as needed [#1269](https://github.com/codex-team/editor.js/issues/1269)
```
Comment (neSpecc) [inline docs/CHANGELOG.md:9]:
```suggestion
- `Improvements` - Unuseful log about missed i18n section has been removed  [#1269](https://github.com/codex-team/editor.js/issues/1269)
```

File: src/components/i18n/index.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import defaultDictionary from './locales/en/messages.json';
-import * as _ from '../utils';
+/**
+ * For debugging messages
+ *import * as _ from '../utils';
```
Comment (neSpecc) [inline src/components/i18n/index.ts:4]: you can remove this import.

File: src/components/i18n/index.ts [hunk 3]
```diff
@@ -60,9 +63,12 @@ export default class I18n {
   private static _t(namespace: string, dictKey: string): string {
     const section = I18n.getNamespace(namespace);
 
-    if (section === undefined) {
-      _.logLabeled('I18n: section %o was not found in current dictionary', 'log', namespace);
-    }
+    /**
+     * For Console Message to Check Section is defined or not
+     *if (section === undefined) {
```
Comment (neSpecc) [inline src/components/i18n/index.ts:68]:
```suggestion
     * if (section === undefined) {
```

File: src/components/i18n/index.ts [hunk 4]
```diff
@@ -60,9 +63,12 @@ export default class I18n {
...
+     *  _.logLabeled('I18n: section %o was not found in current dictionary', 'log', namespace);
+     *}
```
Comment (neSpecc) [inline src/components/i18n/index.ts:70]:
```suggestion
     * }
```

**Review by neSpecc — APPROVED** [review]

**Review by talyguryn — APPROVED** [review]

**Pull request merged**